public/locations-list-map.js

```diff
@@ -104,6 +104,14 @@ const addDonationSites = () => {
 
     largeDonationElement.innerHTML = $.i18n(countryDonationSites.i18nString, administrativeRegionStringHtml, nationalLinksHtml);
     noDonationsElement.innerHTML = $.i18n(countryConfig.noDonationSitesNearMeI18nString);
+
+    $(largeDonationElement).find('a').click(function(e) {
+      sendEvent('largeDonation', 'click', $(this).attr('href'));
```

Comment:
I assume click handlers still work when you redirect the user to an external url?

Reply:
it does...although thinking about that question raises another one that I don't know the answer to -- is that guaranteed by spec, or is it possible a browser implementation (or weird set of circumstances) would lead to it not executing the new click event? not really anything to worry about here but I am going to look into it.